Makefile

```diff
@@ -118,6 +118,28 @@ iao.owl: build/iao-merged.owl
 	--annotation owl:versionInfo "$(TODAY)" \
 	--output $@
 
+### Test
+#
+# Run tests
+
+# Run a reasoner to find inconsistencies
+.PHONY: reason
+reason: build/iao-merged.owl | build/robot.jar
+	@echo "Running tests"
+	@echo "Run reasoning"
+	$(ROBOT) reason --input $< --reasoner hermit --equivalent-classes-allowed none
+
+# Run robot report
+build/robot-report.tsv: build/iao-merged.owl
+	@echo "Generate robot report"
+	$(ROBOT) report \
+	--input $< \
+	--fail-on none \
```

Comment:
```suggestion
	--fail-on error \
```

Reply:
No point without this.

Reply:
IAO import full version BFO classes-only OWL file. In BFO, the label of IAO_0000118 is out-of-date. So, it will cause error. I'd prefer to generate the report even it contains error(s). So, users can fix it. Will change it when got a new version of BFO.

Reply:
That defeats the purpose of automated CI though - the point is that the action tells you "pass or no pass" so that no errors are merged in!

Reply:
Thanks for the suggestion. I removed the IAO_0000118 alternative label from checking.